From 1450881d8d7674c89d1371ff92477f3e98bb3c9e Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Fri, 29 Oct 2021 08:21:11 -0700
Subject: [PATCH] Fix segfault on missing file

---
 libretro/core/soundplay.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libretro/core/soundplay.c b/libretro/core/soundplay.c
index c406b96..30b1c08 100644
--- a/libretro/core/soundplay.c
+++ b/libretro/core/soundplay.c
@@ -301,6 +301,9 @@ sound_t *syssnd_load(char *name)
    sound_t *s     = malloc(sizeof(sound_t));
    data_file_t *f = data_file_open(name);
 
+   if (!s || !f)
+      return NULL;
+
    data_file_read(f, &head, 1, WAV_HEADER_SIZE);
 
    s->len         = retro_le_to_cpu32(head.Subchunk2Size);
-- 
2.30.2

